core/server/server.js

```diff
@@ -412,7 +412,7 @@ class Server {
 
     if (rasterUrl) {
       // Redirect to the raster server for raster versions of modern badges.
-      camp.route(/\.png$/, (queryParams, match, end, ask) => {
+      camp.route(/^\/((?!img\/)).*\.png$/, (queryParams, match, end, ask) => {
```

Comment:
This is to allow `/img/logo.png` to work without trying to redirect to squint.
This does bring up a slightly wider question though: The way I've done this, I have put the frontend in the root so the compiled website structure (`assets`, `badges`, `community`, `search`, etc) lives in the same namespace as the badge URLs. This is pretty much what we did with gatsby too but because every badge has its own URL for the builder, we could consider putting the frontend in `/docs` or whatever and make `/` a redirect to `/docs`. I think we should really decide this now because once we launch this version of the site there will be a bunch of links out there in the wild which will break if we change our minds. Thoughts?

Reply:
do you have a feel for which may give us more flexibility over the long term? Also, I gather that `/docs` was just an example but given that we have an existing `doc` directory I think we'd want to use something with a more differentiated name for the generated site output

Reply:
So if we followed that suggestion, the site frontend would live at https://shields.io/docs and https://shields.io/ would redirect to https://shields.io/docs

The code (or output) would not need to live in https://github.com/badges/shields/tree/master/docs in the repo though.

Basically the advantage of that is it would get all the frontend routes out of the same namespace we use for the badge routes but at the moment we don't have any conflicts to worry about.

Reply:
I hate trying to mentally parse regexes, but last, semi-pathological question on this.. would this potentially match any cases that _should_ get redirected to squint?

e.g. a repo named `img`, perhaps like https://img.shields.io/github/v/tag/pathological-username/img/some-branch.png

Reply:
If that's not the case and just some bad mental regex parsing and hypothesizing then great, if we think that's an extremely fringe edge case hypothetical that's not worth the gymnastics to try to work around then I'm also good with that; just wanted to think through possibilities.

Reply:
Yeah good question.

`img` would have to be the first "clause", so it would clash if we tried to add a _service_ called `img`, but it is fine if there is a package or repo called `img`, as in your example.

The first part of the regex is:

- `^` - beginning of the string
- `\/` - escaped slash character
- `((?!img\/))` - anything other than `img`